res/values/strings.xml

```diff
@@ -42,19 +43,21 @@ If you decide to keep it disabled, consider enabling “%2$s“ option to improv
     <string name="always_on_bluetooth_scanning_enabled_but_disallowed">Always-on Bluetooth scanning is enabled, but not allowed (no “Nearby devices“ permission).</string>
     <string name="always_on_bluetooth_scanning_disabled">Always-on Bluetooth scanning is disabled.</string>
 
-    <string name="play_store_obb_permission">
-"To install apps that include OBB files (usually the ones that are over 100 MB in size, such as games), grant Play Store the “Files and media“ and fully restart it with the “Force stop“ button."
-    </string>
-    <string name="no_play_games">
-"Many games depend on the Play Games app for features such as progress saving. If it’s absent, they’ll show a misleading “Missing permissions” prompt."
-    </string>
-    <string name="install_play_games">Install Play Games</string>
-
     <string name="push_notifications">Push notifications</string>
     <string name="play_services_no_battery_exemption">
 "When device is idle, push notifications will be delayed to improve battery life.
 To receive them quicker, set “Battery usage“ to “Unrestricted“ in Play services app info."
     </string>
 
     <string name="play_store_pending_user_action_notif">Action required in Play Store</string>
+    <string name="missing_permission">Missing permission</string>
+    <string name="missing_app">Missing app</string>
+    <string name="play_store_missing_obb_permission_notif">Play Store needs “Files and media“ permission to install this app</string>
+
+    <string name="missing_permission_nearby_exposurenotifications">
+"Play services needs “Nearby devices“ permission to enable Exposure Notifications"
```

Comment:
"Play services needs _the_ “Nearby devices“ permission to enable Exposure Notifications"